example/readFromString/CMakeLists.txt

```diff
@@ -1,3 +1,2 @@
-add_executable(readFromString readFromString.cpp)
-target_include_directories(readFromString PUBLIC ${CMAKE_SOURCE_DIR}/include)
-target_link_libraries(readFromString jsoncpp_lib)
\ No newline at end of file
+ADD_EXECUTABLE(readFromString readFromString.cpp)
+TARGET_LINK_LIBRARIES(readFromString PRIVATE jsoncpp::framework)
```

Comment:
```bash
#!/bin/bash

# \author Hans J. Johnson

this_script_name=$(basename $0)
CMTMSG=$(mktemp -q /tmp/${this_script_name}.XXXXXX)

cat > ${CMTMSG} << EOF
STYLE: Convert CMake-language commands to lower case

Ancient CMake versions required upper-case commands.  Later command names
became case-insensitive.  Now the preferred style is lower-case.
EOF

SED_BIN=$(which gsed)
if [[ -z "${SED_BIN}" ]];then
  SED_BIN=sed
fi


#Run the following shell code:
## NOTE: MUST USE GNU compliant version of sed
cmake --help-command-list \
| grep -v "cmake version" \
| while read c; do
    echo 's/\b'"$(echo $c | tr '[:lower:]' '[:upper:]')"'\(\s*\)(/'"$c"'\1(/g'
  done > /tmp/convert.sed \
&& git ls-files -z -- bootstrap '*.cmake' '*.cmake.in' '*CMakeLists.txt' '*.wrap' \
   | xargs -0 ${SED_BIN} -i -f  /tmp/convert.sed \
&& rm  /tmp/convert.sed

git add -A && git commit --file ${CMTMSG}
```

Reply:
Sir @hjmjohnson , I have already made the preformatting I wanted.